tests/rptest/services/redpanda.py

```diff
@@ -2067,13 +2067,7 @@ def pids(self, node):
         * 'redpanda' started in 'start_redpanda'
         """
         try:
-            proc_name_regex = "redpanda"
-            cmd = f"ps ax | grep -i '{proc_name_regex}' | grep -v grep | awk '{{print $1}}'"
-            pid_arr = [
-                pid for pid in node.account.ssh_capture(
-                    cmd, allow_fail=True, callback=int)
-            ]
-            return pid_arr
+            return [self.redpanda_pid(node)]
```

Comment:
`self.redpanda_pid(node)` may return `None` and  `[None]` is definitely an incorrect thing to return here

Reply:
also redpanda_pid should not treat errors as 404 it's a wrong signal. options: retry or fail

Reply:
I don't want to change the semantics of this method, so will leave the error handling as is

Reply:
I don't want to change the semantics of this method, so will leave the error handling as is